Doc/whatsnew/3.14.rst

```diff
@@ -89,7 +89,7 @@ and improvements in user-friendliness and correctness.
 * :ref:`PEP 750: Template strings <whatsnew314-pep750>`
 * :ref:`PEP 758: Allow except and except* expressions without parentheses <whatsnew314-pep758>`
 * :ref:`PEP 761: Discontinuation of PGP signatures <whatsnew314-no-more-pgp>`
-* :ref:`PEP 765: Disallow return/break/continue that exit a finally block <whatsnew314-pep765>`
+* :ref:`PEP 765: Disallow return/break/continue that exit a finally block <whatsnew314-finally-syntaxwarning>`
```

Comment:
Why not use `-pep765` like the others?

I have previously done things like start at https://docs.python.org/3.14/whatsnew/3.14.html#whatsnew314-pep779 and edit the anchor in the address bar to https://docs.python.org/3.14/whatsnew/3.14.html#whatsnew314-pep765

Reply:
This is the first change, future PRs for the 'New Features' section will continue to make these changes -- to me, it's much better to have a concise description of what it is, especially given that many of the current _New Features_ entries aren't backed by PEPs.